src/Response/StringResponse.php

```diff
@@ -80,12 +92,12 @@ private function __construct()
      * @param string $contentType
      * @return Response
      */
-    private static function createResponse($body, $status, array $headers, $contentType)
+    private static function createResponse($body, $status, array $headers, $contentType = null)
```

Comment:
Keep the original signature, but have the caller send a null value or an empty string.